definitions/ext-fastly-datacenter/definition.yml

```diff
@@ -0,0 +1,33 @@
+domain: EXT
+# Derived from the Fastly log ingestion, a datacenter is a Point of Presence
+# https://docs.fastly.com/en/guides/fastly-pop-locations
+type: FASTLY-DATACENTER
+
+synthesis:
+  name: fastly-datacenter
```

Comment:
Some questions: 

- Are `name` and `identifier` meant to be the same (they use `-` and `_`)
- I assume the `fastly-datacenter` property in this telemetry will be considered unique for a given account, is that right?
- Do you know how long are `fastly-datacenter` values? (this is relevant in order to know if we should encode the identifiers, they would remain untouched in the name.

Reply:
@srvaroa The fastly-datacenter property is unique for a given account. Each Datacenter has a 3 character code that is expressed by Fastly.

Reply:
Fantastic, thanks!